puterbot/record.py

```diff
@@ -668,6 +693,17 @@ def record(
     )
     window_event_writer.start()
 
+    terminate_perf_event = multiprocessing.Event()
```

Comment:
Why not re-use `terminate_event` here?

Reply:
We need a separate multiprocessing.Event() so that the event writers finish putting to perf_q before the performance stat writer terminates. Otherwise we risk losing data and risk a deadlock like #93.